### PR TITLE
Linter addons

### PR DESCRIPTION
**Pull request by ductiletoaster**
This breaks the linters into two separate addons. The goal eventually will be to add another one for es lint that will take over for tsLint.

Follow ups:
1) Add package for eslint
2) If possible use standalone packages instead of process wrapping as the standalone packages gives us more controll and better stability/testing.
3) Document the new addons

### CODE REVIEW

File: packages/pixeloven-storybook/config/package.json [hunk 1]
```diff
@@ -25,8 +25,8 @@
         "css-loader": "3.4.2",
         "deepmerge": "4.2.2",
         "fork-ts-checker-webpack-plugin": "4.1.2",
-        "sass": "1.24.1",
-        "sass-loader": "8.0.0",
+        "sass": "1.26.2",
+        "sass-loader": "8.0.2",
```
Comment (ductiletoaster) [inline packages/pixeloven-storybook/config/package.json:29]: Just doing this to bring it inline with the main webpack build

**Review by jimmynotjim — CHANGES_REQUESTED** [review]

File: packages/pixeloven/cli-core/src/commands/lint/lint.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export default {
+    description: "Linters",
+    name: "lint",
+    run: async () => {
+        // place holder for linting
+        // @todo should scann for linter addons and construct a menu based on some propery of the commands
```
Comment (jimmynotjim) [inline packages/pixeloven/cli-core/src/commands/lint/lint.ts:6]: `scann` => `scan`

File: apps/examples/ssr/src/shared/components/pages/Home/Home.stories.tsx [hunk 3]
```diff
@@ -17,8 +17,8 @@ const match = {
 };
 
 export default {
-    title: "Components/Pages/Home",
     component: Home,
+    title: "Components/Pages/Home",
```
Comment (jimmynotjim) [inline apps/examples/ssr/src/shared/components/pages/Home/Home.stories.tsx:21]: 👏

File: packages/pixeloven/cli-addon-stylelint/src/commands/lint/style.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        const { parameters, pixelOven, print, styleLint } = context;
+        const argList = pixelOven.getArgList("lint", parameters, {
+            offset: 2,
+            type: "withOptions",
+        });
+        const results = await styleLint(argList);
+        statusCode = results.status;
+        if (!statusCode) {
+            print.success(
+                `Success! Your code is beautify just the way it is.\n`,
```
Comment (jimmynotjim) [inline packages/pixeloven/cli-addon-stylelint/src/commands/lint/style.ts:18]: I will never stop laughing at this

File: packages/pixeloven/cli-addon-stylelint/src/types.ts [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import { PixelOvenToolbox } from "@pixeloven/cli";
+
+interface RunResponse {
+    stdout?: Buffer | string;
+    status: number;
+    error?: Error;
```
Comment (jimmynotjim) [inline packages/pixeloven/cli-addon-stylelint/src/types.ts:6]: lol, alpha sorted the others above but not this one ;)

**Review by jimmynotjim — APPROVED** [review]

File: packages/pixeloven/cli-addon-stylelint/src/commands/lint/style.ts (hunk 4 above)
Comment (ductiletoaster) [reply packages/pixeloven/cli-addon-stylelint/src/commands/lint/style.ts:18]: haha

File: packages/pixeloven/cli-addon-stylelint/src/types.ts (hunk 5 above)
Comment (ductiletoaster) [reply packages/pixeloven/cli-addon-stylelint/src/types.ts:6]: Only what the linter tells me ;)

**Pull request merged**